shm/Store.cpp

```diff
@@ -47,7 +47,7 @@ Response Store::buy(const Cargo* const cargo, size_t amount, Player* player) {
     if (!buyedCargoPart) {
```

Comment:
This is the first time I've seen that part of our code and I've realized that we have some language mistakes like that one. Buy in past tense should be "bought". Although this is not part of the commit I would ask to correct it asap - in this PR if possible.